x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/ParsingTests.java

```diff
@@ -93,6 +94,10 @@ public void testInlineCast() throws IOException {
             report.humanReadable(true).prettyPrint();
             report.startObject();
             List<String> namesAndAliases = new ArrayList<>(DataType.namesAndAliases());
+            if (Build.current().isSnapshot() == false) {
```

Comment:
I'd replace this by a check for the specific capability. Otherwise, it's easy to forget to change this piece of code once that geotiles get enabled permanently.

Also applies to the other test fixes.